addons/mail/static/src/core/common/store_service.js

```diff
@@ -231,7 +231,11 @@ export class Store extends BaseStore {
         const ids = Object.keys(this.Thread.records).sort(); // Ensure channels processed in same order.
         for (const id of ids) {
             const thread = this.Thread.records[id];
-            if (thread.model === "discuss.channel" && !thread.isTransient) {
+            if (
+                thread.model === "discuss.channel" &&
+                !thread.isTransient &&
+                thread.fetchChannelInfoState === "fetched"
+            ) {
```

Comment:
This fill conflict with forward-port of https://github.com/odoo/odoo/pull/154170